index.html

```diff
@@ -44,44 +91,139 @@
                         <div class="hp"></div>
                         <div class="hp"></div>
                     </div>
-                    <div class="card"></div>
+                    <div>
+                        <div class="portait-wrap">
+                            <div class="portait"></div>
+                        </div>
+                        <div class="buff-list right">
+                            <div class="buff-wrap">
+                                <div class="buff buff-1"></div>
```

Comment:
can you avoid hard numeration in classes? looks like we have only five buffs max, no more